pennylane_lightning/core/src/simulators/lightning_tensor/cutn/CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+cmake_minimum_required(VERSION 3.20)
+
+set(LOGO [=[
+░█░░░▀█▀░█▀▀░█░█░▀█▀░█▀█░▀█▀░█▀█░█▀▀░░░░▀█▀░█▀▀░█▀█░█▀▀░█▀█░█▀▄░░░░█▀▀░█░█░▀█▀░█▀█
+░█░░░░█░░█░█░█▀█░░█░░█░█░░█░░█░█░█░█░░░░░█░░█▀▀░█░█░▀▀█░█░█░█▀▄░░░░█░░░█░█░░█░░█░█
+░▀▀▀░▀▀▀░▀▀▀░▀░▀░░▀░░▀░▀░▀▀▀░▀░▀░▀▀▀░▀░░░▀░░▀▀▀░▀░▀░▀▀▀░▀▀▀░▀░▀░▀░░▀▀▀░▀▀▀░░▀░░▀░▀
+]=])
+message(${LOGO})
+
+project(${PL_BACKEND}
+    DESCRIPTION "Lightning-Tensor MPS bindings for PennyLane. Backed by NVIDIA cuQuantum SDK."
+    LANGUAGES CXX C
+)
+
+include("${pennylane_lightning_SOURCE_DIR}/cmake/support_pllgpu.cmake")
+include("${pennylane_lightning_SOURCE_DIR}/cmake/support_pltensorcutn.cmake")
+findCUDATK(lightning_external_libs)
+findCutensornet(lightning_external_libs)
+
+set(LTENSOR_MPS_FILES  MPSCutn.cpp CACHE INTERNAL "" FORCE)
+
+add_library(${PL_TENSOR} STATIC ${LTENSOR_MPS_FILES})
+
+##########################
+## Enforce C++ Standard ##
+##########################
+set(CMAKE_CXX_EXTENSIONS OFF)
+set(CMAKE_CXX_STANDARD 20)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
+
+#########################
+## Set Default Options ##
+#########################
+option(PL_DISABLE_CUDA_SAFETY "Build without CUDA call safety checks" OFF)
+
+target_link_libraries(${PL_TENSOR} PUBLIC  lightning_compile_options
+                                            lightning_external_libs
+                                            lightning_tensor_utils
+                                            lightning_tensor_tensor
```

Comment:
`lightning_tensor_tensor` :thinking: can we change the name to something more explicit here? :)

Reply:
Could the generic tensor stuff also move under [pennylane_lightning/core/src](https://github.com/PennyLaneAI/pennylane-lightning/pull/704/files/27e3dab0134813d43ea0d8c6cdf177d824635439#diff-a4624c2dbdb6493050de25c56c553a5bc021fe16f3c3a759626c699ca17ebace)?

Reply:
The lib name is set as `${PL_BACKEND}_tensor`. Is it more clear?  In order to group related modules together, I'm inclined to keep the tensor stuff in the current directory.